docs/pages/api.rst

```diff
@@ -124,22 +124,8 @@ Signature: ``Collection::range(float $start = 0.0, float $end = INF, float $step
 
 .. code-block:: php
 
-    $fibonacci = static function ($a = 0, $b = 1): array {
-        return [$b, $a + $b];
-    };
-
     $even = Collection::range(0, 20, 2); // [0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20]
 
-Another example
-
-.. code-block:: php
-
-    $even = Collection::unfold(static function ($carry) {return $carry + 2;}, -2);
-    $odd = Collection::unfold(static function ($carry) {return $carry + 2;}, -1);
-    // Is the same as
-    $even = Collection::range(0, \INF, 2);
-    $odd = Collection::range(1, \INF, 2);
-
```

Comment:
these were repeated in both the `range` section and the `unfold` section. I kept it for the latter

Reply:
Fair enough.